files/systemd-sysusers-config/placeholders.conf

```diff
@@ -0,0 +1,18 @@
+# Reserved system users and groups for future use in Endless OS 7.
+#
+# This config defines system users and groups that have static IDs in Endless OS
+# 6, which may own files in /etc or /var of existing Endless OS systems, and so
+# the UIDs and GIDs need to be reserved in EOS7 too.
+#
+# System user config should be installed in the element for the corresponding
+# system service. The users below are special cases where we don't (yet) have
+# an element defined in EOS7.
+
+#Type Name       ID
+g     colord     112
+u!    colord     106:112
+g     kalite     128
+u!    kalite     121:128
```

Comment:
kalite comes from https://github.com/endlessm/eos-kalite.
But, EOS 7 will not include it.  So, good to move the user and group into obsolete system users and groups.

Reply:
Done!